src/components/Homepage/MonthlyGoodExplained.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+	<div class="monthly-good-explained">
+		<div class="row">
+			<h1 class="column small-12 text-center">
+				How does Monthly Good work?
+			</h1>
+			<div class="steps-container">
+				<div class="column small-12 medium-6 large-3 step-wrapper">
+					<span class="number">
+						1
+					</span>
+					<kv-icon name="hp-mg-1-signup" class="mg-image" />
```

Comment:
Since these .svgs are only used on the homepage I think they would be good candidates for loading as inline-svgs rather than adding to the svg sprite. By adding to the sprite that data will be loaded on every page.

There's an example of this inline loading on one of the year-end promo [pages with the "sketchy arrow"](https://github.com/kiva/ui/blob/master/src/pages/Possibility/YearEnd.vue#L71). Basically the .svg file needs to be in assets/inline-svgs and then you import it and use it as if it were a Vue component.

Reply:
I agree with Ryan on this, I think it would be great to be able to move away from the SVG sprite going forward

Reply:
I've successfully transitioned to pulling this svg inline. However, when I attempt to load the homepage up now, it's throwing an error for each of the svgs i'm trying to import. 
"You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file."
- I'm wondering if I have to export these svgs differently because the 'sketchy-arrow.svg' file looks pretty different than any of my svg files ie: 'hp-mg-1-signup.svg'. 
- I also don't see any type of additional loader that was used for the 'sketchy-arrow.svg' file which leads me to believe i'll have to re-export these svg in a different way... 
Would love any tips or suggestions related to a fix for this.

Reply:
@BoulderBrains the svgs have to be inside of the inline-svg folder so that they get processed by the correct webpack loader

Reply:
Thank you @eddieferrer

Reply:
@ryan-ludwig pointed me to lines 89 -> 95 in webpack.base.conf.js which explained the additional loader required to load these svgs inline. Thanks for that @ryan-ludwig